test/rootchain_helpers.js

```diff
@@ -0,0 +1,225 @@
+let RLP = require('rlp');
+
+/*
+ How to avoid using try/catch blocks with promises' that could fail using async/await
+ - https://blog.grossman.io/how-to-write-async-await-without-try-catch-blocks-in-javascript/
+ */
+let to = function(promise) {
```

Comment:
Possibly change the name of this function

Reply:
I think it is fine. Open to changing it though

Reply:
would `promise` work?

Reply:
I vote for keeping it. it's concise

Reply:
is it just used in the context of catching errors? it's concise, but someone who is reading through our tests will have to look at the function definition to figure out what to does.

Ideally, people should read the function and know what it does.

Reply:
remove the definition from this file